keyboards/tempo_turtle/bradpad/keymaps/4brad/keymap.c

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2021 v3ritas <v3ritas@tempoturtle.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "midi.h"
+#include "process_midi.h"
```

Comment:
These shouldn't be needed. 
```suggestion
```

Reply:
Double checking this, think it might be needed because I'm accessing the midi-device code in that keymap.

Reply:
Works without imports, good to know.

Reply:
Yeah, most of the stuff like this is included "upstream".  Usually in quantum.h, which is included by the keyboard's header file.